core/src/main/java/ai/timefold/solver/core/api/score/buildin/hardmediumsoftbigdecimal/HardMediumSoftBigDecimalScore.java

```diff
@@ -132,6 +137,7 @@ public static HardMediumSoftBigDecimalScore ofSoft(BigDecimal softScore) {
      * timefold-solver-jpa, timefold-solver-jackson, timefold-solver-jaxb, ...
      */
     @SuppressWarnings("unused")
+    // TODO: params default or null?
```

Comment:
Well spotted. In the interest of getting rid of null, default values (`BigDecimal.ZERO`) would be preferred.